.cicdtemplate/.codemagic/codemagic.yaml

```diff
@@ -22,9 +22,11 @@ definitions:
       script: ./gradlew detekt
     - &unit_test
       name: Run unit tests
-      script: ./gradlew koverMergedReport
+      script: |
+        ./gradlew koverHtmlReport
+        ./gradlew koverXmlReport
```

Comment:
XML report is for Danger to execute. Do we need it in this workflow?

Reply:
You are right @luongvo . I don't see any task running Danger in the workflow. I am not sure why we need `koverMergedReport` initially. I just migrated to the closes thing I can 😆 . Shall we only run HTML report then? What do you think @lydiasama? Do you have any concerns about it?

Reply:
I just followed the previous workflow when initialing this codemagic workflow. 🤔 
BTW, I don’t have any concerns.  Let’s remove the XML report if we don’t need it 😬  @kaungkhantsoe  @luongvo